data/settings.js

```diff
@@ -7509,11 +7517,11 @@ to prefix mailbox names with \`/path/\` or \`~user/\` indicators.`
 	mail_gid: {
 		seealso: [ 'mail_uid' ],
 		values: [ setting_types.STRING, setting_types.UINT ],
-		// TODO: Describe value format (comma-separated list?)
+		default: '*[system dependent]*',
 		text: `
 The system group ID used for accessing mail messages.
 
-Can be either numeric IDs or group names.
+Can be either a numeric IDs or a group names.
 
 If you use multiple values here, userdb can override them by returning the
 gid field.`
```

Comment:
This sentence also doesn't make sense. We have mail_groups setting which can contain many groups, but mail_gid is only one value. And even if there were multiple groups, I don't understand what the sentence is trying to say.